modules/web-console-query-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * web_console/web-console-query-cli.adoc
+
+[id="web-console-query-cli_{context}"]
+= Query image vulnerabilities from the CLI
+Using the `oc` command, you can display information about
+vulnerabilities detected by the Container Security Operator.
+
+.Prerequisites
+* Be running the Container Security Operator on your
+{product-title} instance
+
+.Procedure
+To query for detected container image vulnerabilities, type:
+
+```
+$ oc get vuln --all-namespaces
+NAMESPACE     NAME              AGE
+default       sha256.ca90...    6m56s
+skynet        sha256.ca90...    9m37s
+```
+To display details for a particular vulnerability, identify one of the
```

Comment:
This sentence could be improved. One option might be "To display details for a particular vulnerability, provide the vulnerability name and its namespace to the `oc describe` command.

Reply:
Okay.

Reply:
Done.